wearApp/src/main/java/dev/johnoreilly/confetti/wear/conferences/ConferencesViewModel.kt

```diff
@@ -3,45 +3,31 @@ package dev.johnoreilly.confetti.wear.conferences
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import androidx.work.WorkManager
-import com.apollographql.apollo3.cache.normalized.FetchPolicy
 import dev.johnoreilly.confetti.ConferenceRefresh
 import dev.johnoreilly.confetti.ConfettiRepository
+import dev.johnoreilly.confetti.utils.ClientQuery.toUiState
+import dev.johnoreilly.confetti.utils.QueryResult
 import dev.johnoreilly.confetti.wear.complication.ComplicationUpdater
 import dev.johnoreilly.confetti.wear.tile.TileUpdater
 import kotlinx.coroutines.flow.SharingStarted
 import kotlinx.coroutines.flow.StateFlow
-import kotlinx.coroutines.flow.flow
 import kotlinx.coroutines.flow.stateIn
 import kotlinx.coroutines.launch
 
 class ConferencesViewModel(
     private val tileUpdater: TileUpdater,
     private val complicationUpdater: ComplicationUpdater,
     private val repository: ConfettiRepository,
-    private val workManager: WorkManager,
     private val refresh: ConferenceRefresh
 ) : ViewModel() {
-    val conferenceList: StateFlow<ConferencesUiState> = flow {
-        var hasConferences = false
```

Comment:
My hope is that ViewModels don't need to deal with this complexity, but can instead just us this function with different fetchModes.  I'm also changing this to CacheFirst, which I think we agreed.